### ISSUE TITLE
Do you know how to get installer, if I buy the game from EPIC?

### ISSUE THREAD

**Issue by flier268** [wine-gameservers/arch-wine-fs22#13]
The server require cd key, but i don't know how to get that

**Comment by Toetje585** [comment]
Epic Games is not supported in this image, however In a new upcoming image I will do an attempt to make it work.

**Comment by flier268** [comment]
1. put this file as `run.bat` under `x64` folder
```bat
@echo off
Set UserId=userid@email.com
Set Password=Password0001
cd /D "C:\Program Files\Epic Games\FarmingSimulator22"
FarmingSimulator2022.exe -AUTH_LOGIN=%UserId% -AUTH_PASSWORD="%Password%"
```

2. Edit `dedicatedServer.xml`
```xml
    <game description="Farming Simulator 22" name="FarmingSimulator2022" exe="FarmingSimulator2022.exe">
```
to
```xml
    <game description="Farming Simulator 22" name="FarmingSimulator2022" exe="run.bat">
```

3. Edit Epic setting, set params `-server`

**Comment by compumark** [comment]
@flier268 so you have epic launcher also installed on the windows part via VNC and added the parameters above?

**Comment by flier268** [comment]
No, I do that on Windows. But I wish I could have a Docker image with the
same functionality

compumark ***@***.***> 於 2024年6月8日 週六 上午3:05 寫道：

**Comment by Toetje585** [comment]
@flier268 I have a test image on Discord that does this, however I suggest to wait a bit until I finalyze it.

**Comment by jaimelaborda** [comment]
How can access the Discord? I'm interested in that image.

**Comment by flier268** [comment]
@Toetje585 
Could you edit DOCKERFILE or install.sh, install game and setup?
1. Download game with [Legendary ](https://github.com/derrod/legendary)
2. create run.bat and use password in env
3. create game.xml and set run.bat

By the way, I had a [new project](https://github.com/flier268/EXE-Proxy) that can forward startup parameters to FarmingSimulator2022.exe (ex: EPIC username and password), so you can set game.xml to
```xml
    <game description="Farming Simulator 22" name="FarmingSimulator2022" exe="EXE Proxy.exe">
```
```json
{
  "FileName": "C:\\Program Files\\Epic Games\\FarmingSimulator22\\FarmingSimulator2022.exe",
  "Arguments": "-AUTH_LOGIN=abc@gmail.com -AUTH_PASSWORD=\"password\"",
}
```

**Comment by Bl4ckBirb** [comment]
For anyone else stumbling upon this years later while trying to make this project work using the Epic Games version of FS22 - I created a fork of this project which does exactly that: https://github.com/Bl4ckBirb/arch-wine-fs22-epic
